CHANGES.md

```diff
@@ -25,6 +25,8 @@ To be released.
         `IImmutableDictionary<Address, IValue>` (was `AddressStateMap`).
      -  `IStore.SetBlockStates()` method became to take
         `IImmutableDictionary<Address, IValue>` instead of `AddressStateMap`.
+ -  Added the `genesisBlock` parameter to
+    `BlockChain<T>(IBlockPolicy<T>, IStore)` constructor.  [[#688]]
```

Comment:
As we have only one public constructor for `BlockChain<T>`, it would be fine if we omit the parameters.

```suggestion
    `BlockChain<T>()` constructor.  [[#688]]
```